MovieKuno/Tab Pages/Home Views/RowContainerView.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  RowContainerView.swift
+//  MovieKuno
+//
+//  Created by Leah Joy Ylaya on 1/21/21.
+//
+
+import Foundation
+import SwiftUI
+
+struct RowContainerView: View {
+    var rowTitle: String
+    var movies: Movies
+    
+    var body: some View {
+        ZStack {
+            Color(Asset.background.color)
+                .ignoresSafeArea()
+            VStack(alignment: .leading) {
+                setupRowHeader()
+                ScrollView(.horizontal) {
+                    setupMoviesStack()
+                }
+            }
+        }
+    }
+    
+    fileprivate func setupRowHeader() -> some View {
```

Comment:
this is not a setup, since it returns a view, so maybe you're goal was to create a custom view? maybe create a new struct instead for this view?